README.md

```diff
@@ -1,2 +1,39 @@
-# frontent-homework
-Vilantis frontend homework task
+# Vilantis frontend homework task
+
+Implement a small web site using any frontend technology you like (preferably React.js or Vue.js) in compliance with the requirements below. Use any additional software necessary. Format your code according to the recommendations of your chosen technology. Add a Readme file with the instructions how to set up and start your website. Upload your solution to a public repo on GitHub and send us a link to it.
+
+Implementation of all the [required] lines is necessary for passing the test. Implementation of the non-required lines gives bonus points. Feel free to come up with and implement your own ideas in addition to, or instead of, the optional requirements.
+
+While working on this assignment, your are welcome to have fun and show off your strengths.
+
+## Required requirements
+- [ ] Allow the user to select multiple `.csv` files (file example is shown below)
+- [ ] List all selected files together with their filenames and line counts
+- [ ] Add an option to remove any file from the list
+- [ ] Show the average of age of all the users
```

Comment:
There is an immediate switch of context that makes this a bit confusing - users were never mentioned before in the task nor that that's what will appear in the input file.

We can either:
Switch the order of lines 13 and 14, specifically mention `that collects users from .csv files` in the latter.
or
Show the .csv file before the requirements, i.e.:
```The system will allow uploading .csv files of this format:
<example>
```